components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/validators/DefaultOAuth2TokenValidatorTest.java

```diff
@@ -54,6 +59,10 @@ public void setUp() throws Exception {
         oAuth2TokenValidationMessageContext =
                 new OAuth2TokenValidationMessageContext
                         (oAuth2TokenValidationRequestDTO, oAuth2TokenValidationResponseDTO);
+
+        AccessTokenDO accessTokenDO = new AccessTokenDO();
+        accessTokenDO.setConsumerKey("consumer-key");
```

Comment:
Define a constant for consumer-key

Reply:
fixed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/942/commits/ca4690d734709b108ef2df33a69ae3c9242febc8